.github/actions/workflow-vars/action.yml

```diff
@@ -79,11 +98,13 @@ runs:
       fi
       # The var is needed in some non-shell steps.
       echo "ingress-host=${TEST_INGRESS_HOST}" | tee -a $GITHUB_OUTPUT
-    # Avoid confusion about the chart version since we only change the version during the release process
-    # as the "version" field in "Chart.yaml" file doesn't reflect the changes since the latest release.
+      echo "TEST_INGRESS_HOST=${TEST_INGRESS_HOST}" | tee -a $GITHUB_ENV
+
+  # Avoid confusion about the chart version since we only change the version during the release process
+  # as the "version" field in "Chart.yaml" file doesn't reflect the changes since the latest release.
   - name: Set chart version
     shell: bash
     # TODO: Use the branch name dynamically as part of the version when it's supported in GitHub:
     # https://github.com/orgs/community/discussions/70343
     run: |
-      yq -i '.version = "0.0.0-ci-snapshot"' charts/camunda-platform/Chart.yaml
+      yq -i '.version = "0.0.0-ci-snapshot"' charts/camunda-platform/Chart.yaml
```

Comment:
Interesting, no linting in place.
A new line is missing but just personal preference.